pkgs/tools/misc/atuin/default.nix

```
@@ -0,0 +1,23 @@
+{ lib, rustPlatform, fetchFromGitHub }:
+
+with rustPlatform;
+buildRustPackage rec {
```

One input per line makes reviews in the future easier.

```suggestion
{ lib
, rustPlatform
, fetchFromGitHub
}:

rustPlatform.buildRustPackage  rec {
```